packages/webdriverio/src/commands/element/getCSSProperty.ts

```diff
@@ -62,17 +68,33 @@ import { parseCSS } from '../../utils'
         //         value: 100
         //     }
         // }
+
+        var height = await elem.getCSSProperty('height', '::after');
+        console.log(height);
+        // outputs the following:
+        // {
+        //     property: 'height',
+        //     value: '1px',
+        //     parsed: {
+        //         type: 'number',
+        //         string: '1px',
+        //         unit: 'px',
+        //         value: 1
+        //     }
+        // }
     })
  * </example>
  *
  * @alias element.getCSSProperty
  * @param  {String}      cssProperty css property name
+ * @param  {?String}      [pseudoElement] the pseudo-element to match. Omitted (or null) for real elements
  * @return {CSSProperty}             The specified css of the element
  *
  */
 export default async function getCSSProperty (
     this: WebdriverIO.Element,
-    cssProperty: string
+    cssProperty: string,
+    pseudoElement?: string | null
```

Comment:
Let's make the second argument and object as it allows us to easier extend it moving on with other parameters. Also, let's ensure to have a strict typing. There are just a handful of possible pseudo elements defined, so rather than defining it as `string` we should define it it as a list of strings, e.g. `'::after' | ...`

Reply:
> Let's make the second argument and object

Do you mean...

```ts
export default async function getCSSProperty (
    this: WebdriverIO.Element,
    { cssProperty, pseudoElement }: {cssProperty: string, pseudoElement?: string | null }
) {
```

... or

```ts
export default async function getCSSProperty (
    this: WebdriverIO.Element,
    cssProperty: string,
    { pseudoElement } : { pseudoElement?: string | null }
) {
```